### PR TITLE
SOAR_3246_Document_Maximum_Size

### PR DESCRIPTION
**Pull request by emilykim-r7**
# Description

Describe the proposed changes:

  - Add line into help.md documenting what the maximum size is for AttackerKB
  - Change example fields for "page" in plugin.spec.yaml

### CODE REVIEW

File: rapid7_attackerkb/help.md [hunk 1]
```diff
@@ -490,6 +491,7 @@ _This plugin does not contain any troubleshooting information._
 
 # Version History
 
+* 1.0.3 - Add maximum size into help.md
```
Comment (jmcadams-r7) [inline rapid7_attackerkb/help.md:494]:
```suggestion
* 1.0.3 - Update to add the maximum size into help documentation
```

The style guide is very prescriptive about log lines.

**Review by jmcadams-r7 — CHANGES_REQUESTED** [review]

**Review by ncramer-r7 — CHANGES_REQUESTED** [review]
Does this refer to the `size` attribute in the Search Assessment and Search Vulnerabilities? Either way, I don't think the update belongs in the Key Features section...maybe instead in the descriptions of the actions that use the size field?

**Review by jmcadams-r7 — APPROVED** [review]

File: rapid7_attackerkb/help.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Description
 
-[Rapid7's AttackerKB](http://attackerkb.com/) is a knowledge base of vulnerabilities and informed opinions on what makes them valuable (or not) targets for exploitation
+[Rapid7's AttackerKB](http://attackerkb.com/) is a knowledge base of vulnerabilities and informed opinions on what makes them valuable (or not) targets for exploitation. The maximum size value is 500.
```
Comment (jschipp-r7) [inline rapid7_attackerkb/help.md:3]: Should be documented in the size input's description

File: rapid7_attackerkb/plugin.spec.yaml [hunk 3]
```diff
@@ -307,7 +307,7 @@ actions:
         description: "Pagination page number, default value is 0"
         required: false
         default: 0
-        example: 0
+        example: 1
       size:
```
Comment (jschipp-r7) [inline rapid7_attackerkb/plugin.spec.yaml:311]: Document in the description field below

**Review by jschipp-r7 — CHANGES_REQUESTED** [review]

**Review by jmcadams-r7 — APPROVED** [review]

**Review by ncramer-r7 — APPROVED** [review]

**Review by jschipp-r7 — APPROVED** [review]

**Pull request merged**